tests/test_parser.py

```diff
@@ -15,6 +15,7 @@ def test_parser_username_link(self):
             api=discourse_api,
             index_topic_id=1,
             url_prefix="/",
+            limit_redirects_to_url_prefix=False,
```

Comment:
I have a few wonderings:

- Should a limit on redirect targets perhaps be the default behaviour for all documentation?
- Or, would this be better as a `redirect_prefix` setting, so you can set the allowed prefix?
- Or should we combine the two, with the default being restrictive but overrideable with a different prefix?

Really the only way to answer this is to do a quick audit of what sorts of redirects currently exist within documentation sets. Do people actually redirect out of the domain, or out of the docs area, in practice?